dask/dataframe/io/parquet.py

```diff
@@ -188,12 +188,39 @@ def _read_fastparquet(fs, fs_token, paths, columns=None, filters=None,
             return _read_fp_multifile(fs, fs_token, paths, columns=None,
                                       categories=None, index=None)
     else:
+        pf = None
         try:
+            # open directory with a metadata file
             pf = fastparquet.ParquetFile(paths[0] + fs.sep + '_metadata',
                                          open_with=fs.open,
                                          sep=fs.sep)
         except Exception:
-            pf = fastparquet.ParquetFile(paths[0], open_with=fs.open, sep=fs.sep)
+            pass
+        if pf is None:
+            # open single file, which may or may not be a metadata file
+            try:
+                pf = fastparquet.ParquetFile(paths[0], open_with=fs.open,
+                                             sep=fs.sep)
+            except Exception:
+                pass
+        if pf is None:
+            # open directory without metadata file, scan for al parq data files
+            try:
+                path = paths[0].rstrip('/')
+                paths = (fs.glob(path + '/*.parq*')
+                         + fs.glob(path + '/*/*.parq*')
+                         + fs.glob(path + '/*/*/*.parq*'))
```

Comment:
Is there possibly more nesting here?  Also, is `.parq` standard or just a convention?